43/README.md

```diff
@@ -235,24 +233,26 @@ ZRE uses UDP IPv4 *beacon* broadcasts to discover nodes and track their presence
 * Each ZRE node SHALL broadcast, at regular intervals, a UDP beacon that identifies itself to any listening nodes on the network.
 * When a ZRE node receives a beacon from a node that it does not already know about, it SHALL consider this to be a new peer.
 * When a ZRE node stops receiving beacons from a peer that it knows about, after a certain interval it SHALL consider this peer to be disconnected or dead.
+* A ZRE node configured with CURVE security SHOULD ignore beacons that do not contain a valid 32-octet CURVE key.
```

Comment:
already done, for all the same reasons :)

in the branch i'm working out of- i did this [as the implementor], and it works OK (eg: you set a public key, it does the right thing). just delineating between that and the  protocol itself (eg: if someone else implemented the protocol outside of the C lib).